src/components/Menus/__tests__/DropdownMenu.spec.js

```diff
@@ -34,6 +34,36 @@ it('renders DropdownMenu with icons and trigger correctly', () => {
   expect(tree).toMatchSnapshot()
 })
 
+it('renders DropdownMenu in open state correctly', () => {
+  const wrapper = mount(
+    <StyleRoot>
+      <div>
+        <DropdownMenu
+          menuContainerOpenStyle={{ zIndex: 1234 }}
+          triggerElement={<Button> Share </Button>}
+        >
+          <MenuItem label="Share via Email" value="email" leftIcon="emailFilled" />
+          <MenuItem
+            label="Share via Facebook"
+            value="facebook"
+            leftIcon="facebookFilled"
+            leftIconStyles={{ color: '#3c5a99' }}
+          />
+          <MenuItem
+            label="Share via Twitter"
+            value="twitter"
+            leftIcon="twitterFilled"
+            leftIconStyles={{ color: '#1da1f2' }}
+          />
+        </DropdownMenu>
+      </div>
+    </StyleRoot>
+  )
+
+  wrapper.find('button').simulate('click')
+  expect(wrapper).toMatchSnapshot()
```

Comment:
the test says "renders dropdownmenu in open state correctly" 

i don't see where we validate that it's open. should we incluee that in this test? the snapshot alone feels somewhat useless. like we're just trying to get coverage.

Reply:
For dropdownmenu, open and close is only a difference in the value of z-inde on one of the child components, so it is captured in this snapshot test.  I can make a more explicit test that asserts the z-index value of that component.

Reply:
Made the tests more specific.